qa/integration/specs/logstash_to_logstash_spec.rb

```diff
@@ -0,0 +1,120 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require_relative '../framework/fixture'
+require_relative '../framework/settings'
+require_relative '../framework/helpers'
+require_relative '../services/logstash_service'
+
+require 'stud/temporary'
+require 'logstash/devutils/rspec/spec_helper'
+
+describe "Logstash to Logstash communication Integration test" do
```

Comment:
review note: we decided to keep LS core integ tests simple and place comprehensive (SSL, multi-host, etc..) cases in the plugin repo.

Reply:
yup! Just simple tests for main tests. More extensive tests will be on the plugin level.